src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -390,6 +390,9 @@
         var X = this.ctrl.__subContext__;
 
         if ( ! X[key] ) {
+          if ( key == 'bareUserDAO' ) {
```

Comment:
can we replace this with the same logic as the 'local' removal below... so something like:
```suggestion
          if ( key.startsWith('bare') ) {
            key = key.replace('bare', '');
            key = key.charAt(0).toLowerCase() + key.slice(1);
```

Reply:
the other instance of the 'bare' DAO is the bareTransactionDAO .. so we could make this a pattern just like the 'local' pattern.

Reply:
Ya I can do that

Reply:
bareTransactionDAO - where @$#@#$ did that come from?